### DISCUSSION TITLE
AttributeError: type object 'Callable' has no attribute '_abc_registry'

### DISCUSSION THREAD

**Discussion by egoiss** [bulletphysics/bullet3#3406] [General]
When i try to put this code:
- python3 -m pybullet_envs.deep_mimic.testrl --arg_file run_humanoid3d_backflip_args.txt

AttributeError: type object 'Callable' has no attribute '_abc_registry'

please let me know how to resolve this. Many thanks!

**Comment by enajx** [comment]
For future references, `pip uninstall typing` seems to solve the issue. It must be some sort of conflict between pybullet and typing.
